test/ui/out/styles/prettify-tomorrow.css

```diff
@@ -0,0 +1,132 @@
+/* Tomorrow Theme */
+/* Original theme - https://github.com/chriskempson/tomorrow-theme */
+/* Pretty printing styles. Used with prettify.js. */
+/* SPAN elements with the classes below are added by prettyprint. */
+/* plain text */
+.pln {
```

Comment:
Gotta say, well done with the documentation and I really like how descriptive these elements are!